### PR TITLE
Fix UI issues in the Console and Recovery Portal apps

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 1]
```diff
@@ -149,8 +127,8 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
                     : null
             }
             {
-                summary?.RoleUserList && summary.RoleUserList instanceof Array &&
-                summary.RoleUserList.length > 0
+                summary?.UserList && summary.UserList instanceof Array &&
+                summary.UserList.length > 0
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:131]:
```suggestion
                summary?.UserList?.length > 0
```

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 2]
```diff
@@ -149,8 +127,8 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
                     : null
             }
             {
-                summary?.RoleUserList && summary.RoleUserList instanceof Array &&
-                summary.RoleUserList.length > 0
+                summary?.UserList && summary.UserList instanceof Array &&
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:130]:
```suggestion
                summary?.UserList && summary?.UserList instanceof Array &&
```

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 3]
```diff
@@ -164,7 +142,7 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
                             <Grid.Column mobile={ 16 } tablet={ 8 } computer={ 8 } textAlign="left">
                                 <Label.Group>
                                     {
-                                        summary.RoleUserList
+                                        summary.UserList
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:145]:
```suggestion
                                        summary?.UserList
```

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 4]
```diff
@@ -183,6 +161,42 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
                     )
                     : null
             }
+            {
+                summary?.GroupList && summary.GroupList instanceof Array &&
+                    summary.GroupList.length > 0
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:166]:
```suggestion
                    summary?.GroupList?.length > 0
```

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 5]
```diff
@@ -183,6 +161,42 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
                     )
                     : null
             }
+            {
+                summary?.GroupList && summary.GroupList instanceof Array &&
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:165]:
```suggestion
                summary?.GroupList && summary?.GroupList instanceof Array &&
```

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 6]
```diff
@@ -183,6 +161,42 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
...
+                                    data-testid={ `${ testId }-users-label` }
+                                    className="label"
+                                >
+                                    { t("adminPortal:components.roles.addRoleWizard.summary.labels.groups") }
+                                </div>
+                            </Grid.Column>
+                            <Grid.Column mobile={ 16 } tablet={ 8 } computer={ 8 } textAlign="left">
+                                <Label.Group>
+                                    {
+                                        summary.GroupList
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:180]:
```suggestion
                                        summary?.GroupList
```

File: apps/console/src/features/roles/components/wizard/role-sumary.tsx [hunk 7]
```diff
@@ -183,6 +161,42 @@ export const CreateRoleSummary: FunctionComponent<AddUserWizardSummaryProps> = (
...
+                                            .map((group, index) => (
+                                                <Label
+                                                    data-testid={
+                                                        `${ testId }-permissions-${ index }-label`
+                                                    }
+                                                    key={ index }
+                                                    basic
+                                                    circular
+                                                >
+                                                    { group.displayName }
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-sumary.tsx:190]:
```suggestion
                                                    { group?.displayName }
```

File: apps/console/src/features/users/components/add-user.tsx [hunk 8]
```diff
@@ -324,22 +246,33 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
                                 showPassword={ t("common:showPassword") }
                                 type="password"
                                 value={ isPasswordGenerated ? randomPassword : initialValues?.newPassword }
-                                listen={ handlePasswordChange }
-                                loading={ isPasswordRegExLoading }
-                                validation={ (value: string, validation: Validation) => {
-                                    if (!isPasswordPatternValid) {
+                                validation={ async (value: string, validation: Validation) => {
+                                    let passwordRegex = "";
+                                    if (userStore !== "primary") {
```
Comment (NipuniBhagya) [inline apps/console/src/features/users/components/add-user.tsx:251]: There's a constant for this in user store constants in the core module. Let's use that here.

**Review by NipuniBhagya — CHANGES_REQUESTED** [review]

File: apps/console/src/features/users/components/add-user.tsx (hunk 8 above)
Comment (thivi) [reply apps/console/src/features/users/components/add-user.tsx]: Resolved by ac16628

File: apps/myaccount/src/components/change-password/change-password.tsx [hunk 9]
```diff
@@ -262,10 +262,11 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
                             setPassword(values.get("newPassword").toString());
                         } }
                         validation={ (value: FormValue, validation: Validation) => {
-                            if (passwordScore < 3) {
+                            // Disable password strength validation
```
Comment (jeradrutnam) [inline apps/myaccount/src/components/change-password/change-password.tsx:265]: Why is it removed? No longer required? If so shall we remove the code out?

File: apps/console/src/features/users/components/add-user.tsx [hunk 10]
```diff
@@ -324,22 +247,33 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
...
+
+                                    if(!validateInputAgainstRegEx(value, passwordRegex)){
                                         validation.isValid = false;
                                         validation.errorMessages.push( t("adminPortal:components.user.forms." +
-                                            "addUserFor1m.inputs.newPassword.validations.regExViolation") );
+                                            "addUserForm.inputs.newPassword.validations.regExViolation") );
                                     }
 
-                                    if (passwordScore < 3) {
+                                    // Password strength validation is disabled
```
Comment (jeradrutnam) [inline apps/console/src/features/users/components/add-user.tsx:268]: Why is it removed? No longer required? If so shall we remove the code out?

Comment (thivi) [reply apps/console/src/features/users/components/add-user.tsx]: Resolved by ff6c234

File: apps/myaccount/src/components/change-password/change-password.tsx (hunk 9 above)
Comment (thivi) [reply apps/myaccount/src/components/change-password/change-password.tsx]: Resolved by ff6c234b

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**